peps/pep-0744.rst

```diff
@@ -133,12 +133,20 @@ Like the rest of the interpreter, the JIT compiler is generated at build time,
 and has no runtime dependencies. It supports a wide range of platforms (see the
 `Support`_ section below), and has comparatively low maintenance burden. In all,
 the current implementation is made up of about 900 lines of build-time Python
-code and 500 lines of runtime C code.
+code and 500 lines of runtime C code. The current approach is optimized for 
+runtime compilation and would likely not work well for ahead-of-time compilation. 
+This is not a direction currently being explored or pursued.
 
 Specification
 =============
 
-The JIT will become non-experimental once all of the following conditions are
+Before discussing the requirements that need to be met before the JIT can be
+considered non-experimental, it is important to clarify that it will always be
```

Comment:
"always" is a long time, do we really want to guarantee the JIT will never become important enough to be required?

Reply:
If we do that, we can write a PEP. I think "always" is a fine promise to put people at ease about waking up one day to it suddenly being required.